.circleci/config.yml

```diff
@@ -28,7 +28,7 @@ jobs:
               sudo pip install virtualenv
               virtualenv venv
               . venv/bin/activate
-              pip install -r $REQUIREMENTS_FILE
+              pip install -r $REQUIREMENTS_FILE --force-reinstall
```

Comment:
@T4rk1n I've already merged this but... was `--force-reinstall` just for debugging or do we need it going forward if we're using git branches in requirements files?